src/wp-includes/formatting.php

```diff
@@ -3115,6 +3119,7 @@ function make_clickable( $text ) {
 					)*
 				)
 				(\)?)                                          # 3: Trailing closing parenthesis (for parenthesis balancing post processing).
+				(\\.\\w{2,6})?               # Allowing file extensions (e.g., .jpg, .png).
```

Comment:
The inline comment will need alignment with the other inline comments, and likely needs to be labeled as `# 4:`